### PR TITLE
Add SkippedTests for TestAction

### PR DESCRIPTION
**Pull request by dp221125**
[Dicussions > Q&A](https://github.com/tuist/tuist/discussions/5703)

### Short description 📝

> Describe here the purpose of your PR.

When create a TestAction in Scheme, it receives TestIdentifiers that skip specific XCTestCases.
In Xcode, To implement the same role as the one pictured below.
image

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally (see [the documentation](https://docs.tuist.io/contributors/testing-strategy) for reference).

When creating Scheme, pass in the skippedTestIdentifiers as shown in the photo below.
image

and, next tuist generate.
and if you look at the test list of that Scheme, you will see that it is disabled, as shown in the photo below.
image

Finally, when we actually run the test, that test case will not be executed.
image

### Contributor checklist ✅

- [x] The code has been linted using run `mise run lint:fix`
- [x] The change is tested via unit testing or acceptance testing, or both
- [x] The title of the PR is formulated in a way that is usable as a changelog entry
- [ ] In case the PR introduces changes that affect users, the documentation has been updated

### Reviewer checklist ✅

- [ ] The code architecture and patterns are consistent with the rest of the codebase
- [ ] Reviewer has checked that, if needed, the PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, and the title is usable as a changelog entry

### CODE REVIEW

File: Sources/ProjectDescription/TestAction.swift [hunk 1]
```diff
@@ -35,6 +35,9 @@ public struct TestAction: Equatable, Codable {
     /// List of diagnostics options to set to the action.
     public var diagnosticsOptions: [SchemeDiagnosticsOption]
 
+    /// List of testIdentifiers to skip to the test
+    public var skippedTestIdentifiers: [String]?
```
Comment (pepicrft) [inline Sources/ProjectDescription/TestAction.swift:39]: What about renaming this one to `skippedTests` to align with the property in `XcodeProj`?

**Review by pepicrft — APPROVED** [review]

**Comment by pepicrft** [comment]
@all-contributors add @dp221125 for code

**Comment by pepicrft** [comment]
Thanks a lot for this contribution @dp221125 👏🏼

File: Sources/ProjectDescription/TestAction.swift (hunk 1 above)
Comment (dp221125) [reply Sources/ProjectDescription/TestAction.swift]: Thank you. I applied the suggestion. df33376

**Pull request merged**